File: Day11/r_demo/mapper.r
#!/usr/bin/Rscript

# THIS IS MAPPER WITH RSCRIPT

input = file('stdin', open = 'r')
while (length(line <- readLines(input, n = 1, warn=FALSE)) > 0){
	words = unlist(strsplit(line, "[[:space:]]+"))
	for (word in words)
		cat(word,"\t1\n", sep='')
}
close(input)

File: Day11/map_only_fb_most_haha/mapper.py
#!/usr/bin/python3

import sys
max_haha = []

flag = 0
for line in sys.stdin:         
    while(flag):
        words = line.split(',')
        max_haha.append(int(words[9]))
        break;
    flag = 1;

print(words[2])
print(max(max_haha))
#3/13/2018 7:07